### DISCUSSION TITLE
can I use a fork of the staking-contracts repo for commercial applications?

### DISCUSSION THREAD

**Discussion by ghost** [0xPolygon/polygon-edge#906] [General]
Good evening, I'd like to fork the staking contracts repo, to set the number of coins required to stake at 5 million. Is there a license required for a commercial project, or any credit needed to be given. 

Thanks.

**Comment by laviniat1996** [comment]
Hello @Nexsuperne101 ,

You can use it, as long as you respect [the Polygon Edge license](https://github.com/0xPolygon/polygon-edge/blob/develop/LICENSE).
